src/locales/en/en.json

```diff
@@ -997,9 +997,11 @@
     }
   },
   "shareidentifier": {
-    "title": "Share identifier",
+    "title": "Share connection",
```

Comment:
If this has been renamed on UI and code as Share Connection, then please rename all keys starting from here all the way down to css class names, component names, tests and copies etc.. Thanks @sdisalvo-crd

Reply:
As per our conversation, I left that on purpose because of the current refactoring in progress that will change how identifiers and credentials are currently being used.